components/automate-ui/src/app/modules/team/team-details/team-details.component.spec.ts

```diff
@@ -151,34 +151,35 @@ describe('TeamDetailsComponent', () => {
     expect(store.dispatch).toHaveBeenCalledWith(new GetProjects());
   });
 
-  it('initializes dropdown with those included on the team checked', () => {
-    spyOn(store, 'dispatch').and.callThrough();
-    const teamProjects = ['b-proj', 'd-proj'];
-    const team: Team = { id: targetId, guid: 'any', name: 'any', projects: teamProjects };
-    store.dispatch(new GetTeamSuccess(team));
-    expect(store.dispatch).toHaveBeenCalledWith(new GetProjects());
+  using([
+    ['no projects', []],
+    ['one project', ['proj-one']],
+    ['multiple projects', ['p1', 'p2', 'p3', 'p4']]
+  ], function (description: string, projects: string[]) {
+    it(`initializes dropdown with those included on the team for ${description}`, () => {
+      const team: Team = { id: targetId, guid: 'any', name: 'any', projects };
+      store.dispatch(new GetTeamSuccess(team));
+
+      expect(component.team.projects).toEqual(projects);
+    });
+  });
 
-    const projectList = [
-      genProject('a-proj'),
-      genProject('b-proj'),
-      genProject('c-proj'),
-      genProject('d-proj')
-    ];
-    store.dispatch(new GetProjectsSuccess({ projects: projectList }));
+  using([
+    ['no projects', []],
+    ['one project', ['proj-one']],
+    ['multiple projects', ['p1', 'p2', 'p3', 'p4']]
+  ], function (description: string, projects: string[]) {
+    it(`transfers result from closing dropdown into form for ${description}`, () => {
+      const originalProjects = ['to-be-overwritten'];
+      const team: Team = { id: targetId, guid: 'any', name: 'any', projects: originalProjects };
+      store.dispatch(new GetTeamSuccess(team));
+      expect(component.team.projects).toEqual(originalProjects);
 
-    projectList.forEach(p => {
-      expect(component.projects[p.id].checked).toEqual(teamProjects.includes(p.id));
+      component.onProjectDropdownClosing(projects);
+
+      expect(component.updateForm.controls.projects.value).toEqual(projects);
```

Comment:
Upon closing the dropdown we merely need to ensure that the form has the correct set of projects, as the form fields are used for saving the team.